tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -461,6 +500,16 @@ def get_all_test_cases():
             yield generate_compat_test(cipher=cipher, sig_alg=sig_alg, named_group=named_group,
                                        server=server, client=client)
 
+        # Generate Hello Retry Request  compat test cases
+        for combine_fields, server, client in \
```

Comment:
I am not sure why the approach is different from the one in `get_all_test_cases` and this does not generate all the possible group couples (20 possibilities).
I rather propose:
```
       # Generate Hello Retry Request  compat test cases                       
        for client, server, client_named_group, server_named_group in \         
            itertools.product(CLIENT_CLASSES.keys(),                            
                              SERVER_CLASSES.keys(),                            
                              NAMED_GROUP_IANA_VALUE.keys(),                    
                              NAMED_GROUP_IANA_VALUE.keys()):                   
            if( client_named_group != server_named_group ):                     
                yield generate_hrr_compat_test(client=client, server=server,    
                                               cert_sig_alg="ecdsa_secp256r1_sha256",
                                               client_named_group=client_named_group,
                                               server_named_group=server_named_group)
```

Reply:
Got it, will change back.